backend/typescript/services/implementations/__tests__/volunteerService.test.ts

```diff
@@ -1,16 +1,14 @@
 import { snakeCase } from "lodash";
 import User from "../../../models/user.model";
-import UserService from "../userService";
 
-import { Role, UserDTO, VolunteerDTO } from "../../../types";
+import { Role, Status, VolunteerDTO } from "../../../types";
 
 import testSql from "../../../testUtils/testDb";
 import VolunteerService from "../volunteerService";
 import Volunteer from "../../../models/volunteer.model";
 
 const testUsers = [
```

Comment:
also as an fyi i think i want to move this into ```testUtils``` because the "mock" test results shouldn't be in the same folder as it buffs it up

Reply:
like the number of lines i did this in one of my past prs at least